packages/components/src/components/modals/Modal/Modal.tsx

```diff
@@ -1,39 +1,57 @@
-/* stylelint-disable indentation */
-import { useCallback, useEffect, useState, ReactNode, MouseEvent } from 'react';
+import { useCallback, useState, ReactNode, useEffect } from 'react';
 
-import styled, { css } from 'styled-components';
+import styled, { css, useTheme } from 'styled-components';
+import { useEvent } from 'react-use';
+import { boxShadows, spacings, spacingsPx, typography } from '@trezor/theme';
 
-import { useKeyPress } from '@trezor/react-utils';
-import { useTheme } from '../../utils/hooks';
-import { Icon } from '../assets/Icon/Icon';
-import { H1 } from '../typography/Heading/Heading';
-import { variables } from '../../config';
-import { IconType } from '../../support/types';
-import { Progress } from '../loaders/Progress/Progress';
+import { Icon } from '../../assets/Icon/Icon';
+import { IconType } from '../../../support/types';
+import { Stepper } from '../../loaders/Stepper/Stepper';
 
-const CLOSE_ICON_SIDE = 26;
-const CLOSE_ICON_PADDING = 16;
+const CLOSE_ICON_SIZE = spacings.xxl;
+const CLOSE_ICON_MARGIN = 16;
 const MODAL_CONTENT_ID = 'modal-content';
 
 const ModalPromptContainer = styled.div`
     margin-bottom: 25px;
 `;
 
+const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    border-radius: 16px;
+    /* when theme changes from light to dark */
+    transition: background 0.3s;
+    max-width: 95%;
+    min-width: 305px;
+    max-height: 90vh;
+    width: 680px;
+    background: ${({ theme }) => theme.backgroundSurfaceElevation1};
```

Comment:
```suggestion
    background: ${({ theme }) => theme.backgroundSurfaceElevation2};
```

In design there is `backgroundSurfaceElevation1` but actually it's using hex of `backgroundSurfaceElevation2`.

Reply:
I can't see it :( Also, I am pretty sure it's elevation 1, since it's white and elevations 1 and 3 are white.

Reply:
`backgroundSurfaceElevation1` is referring to hex of `backgroundSurfaceElevation2`:
![6940FA49-BBA1-468D-A52E-483B1A0021C5_4_5005_c](https://github.com/trezor/trezor-suite/assets/66002635/29806958-fa25-4514-a2c8-fadb9d5d3c6b)

Then it's mistake in the design I guess.